app/src/main/java/com/github/se/signify/model/user/User.kt

```diff
@@ -0,0 +1,10 @@
+package com.github.se.signify.model.user
```

Comment:
- Clearly structured data model with appropriate default values, ensuring null-safety and easier Firestore serialization/deserialization.
- Use of emptyList() for friend-related lists avoids null issues.
- The User class seems well-defined and directly integrates into Firestore with minimal additional mapping.